14/umbraco-cms/fundamentals/code/subscribing-to-notifications.md

```diff
@@ -24,11 +34,19 @@ namespace MyProject;
 
 public class LogWhenPublishedHandler : INotificationHandler<ContentPublishedNotification>
 {
-    // Here we'll handle a notification.
+    // Here we will handle a notification.
 }
 ```
 
-However, we have an error and a red squiggly line under our class. This is because the `INotificationHandler` interface expects us to implement a `Handle` method for our notification. Let's create a public method called `Handle` that takes a `ContentPublishedNotification` as a parameter:
+{% endcode %}
+
+## Implement the Handle Method
+
+The `INotificationHandler` interface requires a `Handle` method to be implemented.
+
+1. Implement the `Handle` method, which takes a `ContentPublishedNotification` parameter. This method will contain the custom logic that runs after content is published.
```

Comment:
Instead of having a list with only 1 step, how about this:

```suggestion
Use the code snippet below to implement the `Handle` method, which takes a `ContentPublishedNotification` parameter. This method will contain the custom logic that runs after content is published.
```